SAP/fundamental
# #732 updates colors with latest Fiori 3.0 design specs

Closes sap/fundamental#732

Updates the colors in the SASS maps, test page and docs pages

#### Test

* Check http://localhost:3030/pages/styles
* Check http://localhost:4000/foundation/colors.html
* Check http://localhost:4000/guides/design-system-utilities.html#colors

#### Changelog

**New**

* Adds inverse `action-2` color
* Adds shell group with `shell-1` and `shell-2` colors
* Adds neutral `status-4` color
* Adds `accent-10` color 
* More content on colors docs page

**Changed**

* Color CSS vars now have a namespace prefix, e.g., `--fd-color-action-1`
* Changes several color values esp. accents
* Implements HSL values in SASS maps
* Fixed some outdated content in helpers page

**Removed**

* Removed primary group SASS variable
